e2e/test_bootstrap.sh

```diff
@@ -10,12 +10,14 @@
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "$SCRIPTDIR/common.sh"
 
+# passing settings to bootstrap but should have 0 effect on it
```

Comment:
I think in our meeting we decided that the default build tag would be 0 for all the built wheels. Currently this applies to bootstrap as well

Reply:
I think bootstrap should run with whatever settings are there, including adding any build tags if needed

Reply:
That will keep the code simpler. It also means if settings change so that a new build tag is needed, that wheel build will be testable with the bootstrap.